tools/scripts/build/download-trending.ts

```diff
@@ -0,0 +1,48 @@
+import { writeFileSync } from 'fs';
+import fetch from 'node-fetch';
+import yaml from 'js-yaml';
+import { trendingSchemaValidator } from '../../../client/i18n/schema/trendingSchema';
+
+import envData from '../../../config/env.json';
+const { clientLocale } = envData;
+
+const cdnUrlCreator = (lang: string) =>
```

Comment:
```suggestion
const createCdnUrl = (lang: string) =>
```
nit it's a function not an object :man_shrugging: